test/orm/test_query.py

```diff
@@ -1225,6 +1225,17 @@ def test_get(self):
         u2 = s.get(User, 7)
         assert u is not u2
 
+    def test_get_one(self):
+        User = self.classes.User
+
+        s = fixture_session()
+        u = s.get_one(User, 7)
+        u2 = s.get_one(User, 7)
+        assert u is u2
+        s.expunge_all()
+        u2 = s.get_one(User, 7)
+        assert u is not u2
```

Comment:
**mike bayer** (zzzeek) wrote:

it's a little awkward this is half tested here and half in test_session, let's move it entirely to one place or the other.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4872